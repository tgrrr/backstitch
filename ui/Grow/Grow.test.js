import React from 'react'
import renderer from 'react-test-renderer'
import { Grow } from '.'

it('Grow: default', () => {
    const component = renderer.create(
        <Grow>
            <>
                Grow
            </>
        </Grow>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
