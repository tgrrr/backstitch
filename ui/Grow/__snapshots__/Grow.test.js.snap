// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grow: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Grow "
    data-testid="Grow"
  >
    Grow
  </div>
</div>
`;
